### PR TITLE
feat: add pronouns

### LINKED ISSUE #36: Pronouns function
### Description

Pronouns is becoming a common field for user profiles

### Proposed solution

Use `rand()` to select from the JSON list of most common pronouns. 

(I have a branch for this ready to go if this feature is wanted)

### Alternatives considered

_No response_

### Do you want to create a pull request?

Yes

### PR DESCRIPTION
**Pull request by theryansmee**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/falso/blob/master/CONTRIBUTING.md#commit
- [] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

Closes #36

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behaviour?
No current behaviour

Issue Number: #36

## What is the new behavior?
Add pronoun function that returns random pronouns from assorted json list

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

## Other information
I haven't added tests are it's just using the `rand()` function with a JSON file

### CODE REVIEW

File: packages/falso/src/lib/pronoun.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "data": ["They/Them", "She/Her", "He/Him", "Ze/Hir"]
```
Comment (NetanelBasal) [inline packages/falso/src/lib/pronoun.json:2]: Why they are not sepreated?

File: packages/falso/src/lib/pronoun.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { FakeOptions, fake } from './core/core';
+import { data } from './pronoun.json';
+
+/**
+ * Generate a random pronoun.
+ *
+ * @category TBD
```
Comment (NetanelBasal) [inline packages/falso/src/lib/pronoun.ts:7]:
```suggestion
 * @category text
```

File: packages/falso/src/lib/pronoun.json (hunk 1 above)
Comment (theryansmee) [reply packages/falso/src/lib/pronoun.json:2]: Sorry, webstorm must have done it when I committed. I'm out but I'll sort it tomorrow morning 😊

Comment (theryansmee) [reply packages/falso/src/lib/pronoun.json:2]: I looks like this is done by a prettier rule that cant be changed. [There's a big discussion about it here.](https://github.com/prettier/prettier-vscode/issues/352)

You can only get around it by being a bit hacky and adding a blank comment to the first line

Comment (NetanelBasal) [reply packages/falso/src/lib/pronoun.json:2]: I'm asking about the `/` - `They/Them`

Comment (theryansmee) [reply packages/falso/src/lib/pronoun.json:2]: Ah sorry I misunderstood what you meant. This is how pronouns are formatted on forms.

So a user might select "She/Her" as their proffered pronouns 😊

Comment (NetanelBasal) [reply packages/falso/src/lib/pronoun.json:2]: I'm not familiar with this concept. Do you have an example?

Comment (theryansmee) [reply packages/falso/src/lib/pronoun.json:2]: So in [this blog post](https://blog.asana.com/2018/05/new-profile-setting-pronouns-asana/) asana are explaining why they've added the pronouns field to their user profile form, and why it's free text.

If you scroll 1/2 way down to the "How to set your pronouns in Asana" section, there's a screen shot 😊

Off the top of my head, Instagram (freetext) and LinkedIn (fixed drop down) also both have similar a field on their user profile form

**Review by NetanelBasal — APPROVED** [review]

**Pull request merged**